stellarphot/photometry/profiles.py

```diff
@@ -0,0 +1,305 @@
+import warnings
+
+import numpy as np
+from astropy.stats import sigma_clipped_stats
+from astropy.nddata import Cutout2D
+from astropy.nddata.utils import NoOverlapError
+from astropy.utils import lazyproperty
+
+from photutils.profiles import RadialProfile, CurveOfGrowth
+from photutils.centroids import centroid_com, centroid_2dg
+
+from stellarphot.photometry import calculate_noise
+
+__all__ = ["find_center", "CenterAndProfile"]
+
+
+def find_center(image, center_guess, cutout_size=30, max_iters=10, match_limit=3):
+    """
+    Find the centroid of a star from an initial guess of its position. Originally
+    written to find star from a mouse click.
+
```

Comment:
You should probably add notes about the algorithm used to identify the centroid and the fact you are masking certain data.